x-pack/winlogbeat/module/security/ingest/security.yml

```diff
@@ -2604,7 +2639,7 @@ processors:
               "4750", "4751", "4752", "4753", "4754", "4755", "4756", "4757",
               "4758", "4759", "4760", "4761", "4762", "4763", "4764", "4767",
               "4781", "4798", "4799", "4817", "4904", "4905", "4907", "4912",
-              "4648"].contains(ctx.event.code)) {
+              "4648", "4797"].contains(ctx.event.code)) {
```

Comment:
Is there a reason not to include the other types here? They also appear to have `SubjectUserName` fields.

Reply:
There was not. I had added them to the related user appending, but I forgot to add them to "Copy Subject User from Event Data" script. Since event 4797 also has a TargetUserName, I also just added that one to the "Copy Target User to Target" script.